src/solver/optimisation/opt_gestion_des_couts_cas_lineaire.cpp

```diff
@@ -26,14 +26,15 @@
 */
 
 #include <yuni/yuni.h>
-#include "opt_structure_probleme_a_resoudre.h"
+#include <antares/study/area/scratchpad.h>
```

Comment:
`#include <antares/study/area/scratchpad.h>` : is there a reason for which we include the scratchpad though we don't need it in this file ?

More generally, we should check if all new header inclusion is necessary.